CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [kirkstone-0.27.2] Q4 2023
+- initramfs:
+  - fixed fsck vfat handling
```

Comment:
how?

Reply:
vfat was not working for u-boot; fixed via added dosfstools.....

Reply:
would be nice what exactly was fixed in the changelog

Reply:
done